src/components/FilterPresetsSelect/FilterPresetItem.tsx

```diff
@@ -49,6 +49,7 @@ export const FilterPresetItem = ({
           >
             <RemoveIcon
               data-test-id="preset-delete-button"
+              // @ts-expect-error - check why style is not accepted in types, but can be used
```

Comment:
tip: This was fixed with last macaw ui version: https://github.com/saleor/macaw-ui/commit/eca6ef88442d76fbbd4b925cc3ae07e3c748b417